Tribler/Test/test_tunnel_community3.py

```diff
@@ -0,0 +1,95 @@
+# Written by Niels Zeilemaker
+# see LICENSE.txt for license information
+import time
+
+# This needs to be imported before anything from tribler so the reactor gets initalied on the right thread
+from Tribler.Test.test_tunnel_base import TestTunnelBase
+
+from Tribler.community.tunnel.hidden_community import HiddenTunnelCommunity
+
+
+class TestTunnelCommunity3(TestTunnelBase):
```

Comment:
Automatic tests are executed in parallel on the turbo jenkins job, by splitting up the tests over multiple files it will speed up the overall time needed for executing all the tests (with a few minutes). Not a nice solution, but it works.